Automattic/newspack-plugin
# Donations – amounts and frequencies customisation

### All Submissions:






### Changes proposed in this Pull Request:



- Adds the ability to customise each default donation amount, as well as toggle default donation frequencies
- Centralises the donations settings – before the settings were stored as product meta when WC was used as platform
- Removes cross-domain tracking for NRH platform (https://github.com/Automattic/newspack-plugin/pull/810), since it requires a full rewrite of the Donate block

Closes https://github.com/Automattic/newspack-blocks/issues/741

### How to test the changes in this Pull Request:

1. On `master`, edit the Donations settings in Reader Revenue wizard and take note of these settings 
2. Insert a couple of Donate blocks – 
  - default
  - manual mode – changing the donation amounts
  - manual mode – changing if tiered  
3. Switch to this branch, observe the blocks render as before
4. Make a donation to verify it works as before 
5. Repeat for the other Reader Revenue platforms

### Other information:
